docs/diagrams/GradeClass.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+skinparam arrowThickness 1.1
+skinparam classBackgroundColor LOGIC_COLOR
+skinparam classAttributeIconSize 0
```

Comment:
Perhaps we could standardise on the format, as yours has the circle icon indicating a Class.